dipy/reconst/ivim.py

```diff
@@ -126,15 +133,43 @@ def f_D_star_error(params, gtab, signal, S0, D):
     return signal - f_D_star_prediction([f, D_star], gtab, S0, D)
 
 
-class IvimModel(ReconstModel):
-    """Ivim model
+def ivim_model_selector(gtab, fit_method='LM', **kwargs):
+    """
+    Selector function to switch between the 2-stage Levenberg-Marquardt based
+    NLLS fitting method (also containing the linear fit): `LM` and the Variable
+    Projections based fitting method: `VarPro`.
+
+    Parameters
+    ----------
+    fit_method : string, optional
+        The value fit_method can either be 'LM' or 'VarPro'.
+        default : LM
+
     """
 
+    if fit_method == 'LM':
+        return IvimModelLM(gtab, **kwargs)
+
+    elif fit_method == 'VarPro':
+        return IvimModelVP(gtab, **kwargs)
+
+    else:
+        print('The fit_method option chosen was not correct. \
```

Comment:
Can you use `warn` instead of  `print`  like [here](https://github.com/nipy/dipy/blob/ebc9c59e280799dde8ba661aa3fe4cb6387e270c/dipy/reconst/ivim.py#L303)

Reply:
Okay!

Reply:
Done!